.github/workflows/build.yml

```diff
@@ -84,54 +87,11 @@ jobs:
       code_coverage: ${{ matrix.code_coverage }}
       static: ${{ matrix.static }}
       unit_tests: true
-      integration_tests: true
```

Comment:
I will remove this from the build config, I think we can always build them, it's not a huge deal